src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/error-handler": "^6.1",
         "symfony/event-dispatcher": "^5.4|^6.0",
         "symfony/http-foundation": "^6.2",
-        "symfony/http-kernel": "^6.2",
+        "symfony/http-kernel": "^6.2.1",
```

Comment:
change needed on this file :)

Reply:
This is need cause of https://github.com/symfony/symfony/pull/48509 . Without it the test fails with deps=low.